### ISSUE TITLE
Calculate Adaptive Layers independently for each object

### ISSUE THREAD

**Issue by Firedrops** [Ultimaker/Cura#10904]
### Is your feature request related to a problem?

When printing multiple objects simultaneously, adaptive layers for each object is applied to all objects, making it essentially print at the finest resolution all the time.

### Describe the solution you'd like

Each object should have adaptive layers independently calculated and applied.

May also require enforced z hop and/or avoid objects during travel options.

### Describe alternatives you've considered

Splitting objects into separate prints, but this ends up taking a lot of extra time anyway due to heating/ABl cycles.

### Affected users and/or printers

Pretty much anyone who uses Adaptive Layers.

**Comment by fvrmr** [comment]
Hi @Firedrops thank you for your feature request.
I will bring this up with the team. Keep you posted!

**Comment by fvrmr** [comment]
I have discussed this request. And this is really complex to implement, because having multiple models with multiple layer heights could cause that the nozzle will bump into another object. That's why I will defer this issue. I hope you understand.

**Comment by Firedrops** [comment]
Of course, as long as the idea's out there, hopefully someone can come up with a good workaround :)

**Comment by mr-september** [comment]
I still think this is worth investigation. Due to issues raised above, implementing it may also need to touch "avoid printed parts when travelling" and/or z-hop algorithms/settings.

**Comment by Firedrops** [comment]
I still think this is worth investigation. Due to issues raised above, implementing it may also need to touch "avoid printed parts when travelling" and/or z-hop algorithms/settings.
